lib/dbservice/schools/enrollment_record.ex

```diff
@@ -7,16 +7,20 @@ defmodule Dbservice.Schools.EnrollmentRecord do
 
   alias Dbservice.Users.Student
   alias Dbservice.Schools.School
+  alias Dbservice.Groups.Group
 
   schema "enrollment_record" do
     field :academic_year, :string
     field :grade, :string
     field :is_current, :boolean, default: false
     field :board_medium, :string
     field :date_of_enrollment, :date
```

Comment:
I don't think we need `date_of_enrollment` anymore, right? Because now we have separate date of school & group enrollment. Previously, we were assuming we only need to capture school enrollment date.

Reply:
Oh yeah!